lib/schemaUtils.js

```diff
@@ -3337,7 +3337,8 @@ module.exports = {
       // go through required schemaPathVariables, and params that aren't found in the given transaction are errors
       _.each(schemaPathVariables, (pathVar) => {
         if (!_.find(determinedPathVariables, (param) => {
-          return param.key === pathVar.name && param._varMatched;
+          // only consider variable matching if url path variables is not allowed
+          return param.key === pathVar.name && (options.allowUrlPathVarMatching || param._varMatched);
```

Comment:
Should `options.allowUrlPathVarMatching` being true resolve the whole operation to true? For example: a case where `options.allowUrlPathVarMatching` is true and `param.key === pathVar.name` is false?

Reply:
@diyaeninan While request matching, matched variables will have the request's path variable key based on what path from schema it matched. As request matching is done using only URL (variables are not considered).

To answer in short, if the request is correctly matched value will be the same.

Reply:
Also if it helps, when the logic of `_varMatched` was introduced, this flow broke. Before that, it handled inline URL variables. PR: https://github.com/postmanlabs/openapi-to-postman/pull/415